pages/index.client.vue

```diff
@@ -0,0 +1,156 @@
+<script setup lang="ts">
+import {
+  AisInstantSearch,
+  AisSearchBox,
+  AisHits,
+  AisRefinementList,
+} from "vue-instantsearch/vue3/es";
+
+const indexName = "test_permit_fees";
+const algolia = useAlgoliaRef();
+
+const transformCategories =
+  (taxonomy_uid: string) =>
+  (
+    items: ToDo[],
```

Comment:
@JordanPisani want to try attempting to create a new type for this? See [types file](https://github.com/Commbocc/Building-Permit-Fee-Estimator/pull/1/files#diff-a150fb53afee945f23f3fe7c176007111f3ba81bb0d188af4077d4a0fc4caa37R1) and a tool like [JSON to Typescript](https://transform.tools/json-to-typescript)